### PR TITLE
ADD NGSIv2 CPr fwd considerations to doc

### PR DESCRIPTION
**Pull request by fgalan**
Don't delete branch after merge.

### CODE REVIEW

File: doc/manuals/user/ngsiv2_implementation_notes.md [hunk 1]
```diff
@@ -73,3 +73,23 @@ However, this may change in the future, as described in https://github.com/telef
 Orion implements a `scope` field in the `POST /v2/op/update` operation (you can see
 [an example in the NGSIv2 walkthrough](walkthrough_apiv2.md#batch-operations)). However, note that this syntax is
 somewhat experimental and it hasn't been consolidated in the NGSIv2 specification.
+
+## NGSIv2 query update forwarding to Context Providers
+
+Context availability management functionality (i.e. operations to register Context Providers) is still to be
+implemented for NGSIv2. However, you can [register providers using NGSIv1 operations](context_providers.md) 
+and have your NGSIv2-based updates and queries being forwarded to Context Providers, getting the response in NGSIv2.
+Internally, the forwarded message and its corresponding response are done using NGSIv1
```
Comment (fgalan) [inline doc/manuals/user/ngsiv2_implementation_notes.md]: "Internally" could be missunderstand as "internal to the CB". Thus, probably is better to say:

"In the CB to CPr communication, the forwarded..."

Comment (kzangeli) [inline doc/manuals/user/ngsiv2_implementation_notes.md]: ok

**Comment by kzangeli** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM

**Pull request merged**